# Don't use GLIBCS private symbol libc_enable_secure.
# Patch taken from Fedoro; See https://bugzilla.redhat.com/show_bug.cgi?id=161536 for details.
# -- amck, 2009-04-16
diff -ruN slang-2.2.2-old/src/slcommon.c slang-2.2.1/src/slcommon.c
--- slang-2.2.2-old/src/slcommon.c	2008-08-08 16:21:27.000000000 +0100
+++ slang-2.2.2/src/slcommon.c	2009-04-16 13:04:02.308826139 +0100
@@ -207,7 +207,7 @@
    return p;
 }
 
-#if !defined(HAVE_ISSETUGID) && defined(__GLIBC__) && (__GLIBC__ >= 2)
+#if 0 && !defined(HAVE_ISSETUGID) && defined(__GLIBC__) && (__GLIBC__ >= 2)
 extern int __libc_enable_secure;
 # define HAVE___LIBC_ENABLE_SECURE 1
 #endif
